src/sidebar/test/annotation-ui-test.js

```diff
@@ -483,28 +491,12 @@ describe('annotationUI', function () {
   });
 
   describe('#updatingAnchorStatus', function () {
-    it("updates the annotation's tag", function () {
```

Comment:
All annotations now have tags as soon as they are added to the store, so `updateAnchorStatus` no longer does what this test checks.